src/main/resources/assets/minebound/lang/en_us.json

```diff
@@ -27,6 +27,7 @@
   "block.minebound.myrial_steel_plate": "Myrial Steel Plates",
 
```

Comment:
Need a `block.minebound.armor_forge` entry here.

Reply:
I'll fix it in [MBM-385](https://minebound.atlassian.net/browse/MBM-385)